app/views/content_items/publication.html.erb

```diff
@@ -12,12 +12,12 @@
 
   <div class="column-two-thirds">
     <%= render 'govuk_component/lead_paragraph', text: @content_item.description %>
-    <%= render 'components/notice', @content_item.withdrawal_notice_component  %>
     <%= render 'shared/history_notice', content_item: @content_item %>
   </div>
 </div>
 
 <%= render 'shared/publisher_metadata_with_logo' %>
+<%= render 'components/notice', @content_item.withdrawal_notice_component  %>
```

Comment:
We need to check with @miaallers about full width vs. 2/3rds width for these notices.